src/modules/BootSupportCheck.rb

```diff
@@ -185,12 +185,12 @@ def check_zipl_part
       boot_part = Storage.GetEntryForMountpoint("/boot") if boot_part.empty?
       boot_part = Storage.GetEntryForMountpoint("/") if boot_part.empty?
 
-      if [:ext2, :ext3, :ext4].include? boot_part["used_fs"]
-        return true
-      else
+      if ![:ext2, :ext3, :ext4].include? boot_part["used_fs"]
```

Comment:
NP: Here I'd fix it the other way (less indentation, no negation in the condition):

```ruby
return true if [:ext2, :ext3, :ext4].include? boot_part["used_fs"]

add_new_problem(_("Missing ext partition for booting. Cannot install boot code."))
false
```